client/my-sites/patterns/pages/category/index.tsx

```diff
@@ -91,10 +91,25 @@ export const PatternsCategoryPage = ( {
 		return {
 			id: category.name || '',
 			label: category.label,
-			link: getCategoryUrlPath( category.name, patternTypeFilterFallback, false ),
+			link:
+				getCategoryUrlPath( category.name, patternTypeFilterFallback, false ) +
+				( isGridView ? '?grid=1' : '' ),
 		};
 	} );
 
+	const handleSettingView = ( value: 'grid' | 'list' ) => {
```

Comment:
```suggestion
	const onChangeViewMode = ( value: 'grid' | 'list' ) => {
```
Not a blocker, but we could rename this to `on...` for consistency.

Reply:
It’s recommended approach by React, so I would keep it to follow the rules of the creators